controllers/src/main/scala/cromwell/pipeline/controller/ProjectFileController.scala

```diff
@@ -65,9 +66,12 @@ class ProjectFileController(wdlService: ProjectFileService)(implicit val executi
       path("files" / "configurations") {
         post {
```

Comment:
Shouldn't we use `get` instead? We are not actually posting anything, we just want to get the configuration of the existing file.

Reply:
It's not my changing, so maybe it's out of scope this PR?

Reply:
It **is** your change. You've significantly changed this endpoint and now it works differently and accept other parameters. It's reasonable to change HTTP method to a more suitable one.

Reply:
Got it thanks.
Fixed.